docs/blog/2018-01-18-how-boston-gov-used-gatsby-to-be-selected-as-an-amazon-hq2-candidate-city/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How Boston.gov used Gatsby to be selected as an Amazon HQ2 candidate city
+date: "2018-01-18"
+author: "Sam Bhagwat"
+---
+
+[Boston.gov](http://boston.gov) is a small team with a huge responsibility: ten people tasked with the digital presence of New England’s most iconic city.
+
+Today, the team and the city scored a huge victory when Amazon announced that Boston was [selected as one among 20 candidate cities](https://www.amazon.com/b?node=17044620011) for its North America HQ2.
+
+![Amazon Boston homepage](./amazon-boston.jpg "Boston city")
+
+When Amazon announced in September that it was looking to build a new headquarters,  bringing 50,000 jobs and billions of investment to the chosen city, Boston’s city government jumped to throw their hat in the ring.
+
+As a technology hub, the city wanted to put their best digital foot forward, so they turned to the Boston.gov team to build a website as the city’s application — [amazon.boston.gov](http://amazon.boston.gov) .
```

Comment:
Curious whether capitalizing Amazon in the URL would make sense. I googled this and many people argue about whether to capitalize proper nouns in URLs haha. Sounds like it doesn't really matter. It just caught my eye since normally Amazon and Boston are capitalized. It should probably stay how it is in lower-case, but something for me to figure out in the future.